### ISSUE TITLE
Windows 10, Putty 0.73, Font: SourceCode Pro, UTF-8, connected to a Fedora 38

### ISSUE THREAD

**Issue by patrickbkr** [japhb/Terminal-Tests#9]
![grafik](https://user-images.githubusercontent.com/6307078/235869868-517b3b51-7df8-435e-a330-c50219c68548.png)

**Comment by japhb** [comment]
Thank you again for the screenshot!

This one at least looks somewhat more reasonable than the previous ones you posted, though this is the first case I've seen where ordinal direction double arrows are available but diagonals are not -- and also the first case of skin tone fallbacks showing skin tone swatches for *some* of the emoji and not others.  It almost seems to be a pattern though, where every third emoji shows the swatch.  Italic and bright attributes are missing; that's something I usually only see on Mac terminals.

May I use this screenshot in the docs?

**Comment by patrickbkr** [comment]
Yes, you may use the screenshot in the docs. Just as the previous screenshots, the Windows didn't have UTF-8 enabled system wide.

**Comment by patrickbkr** [comment]
Having the [default system locale set to use UTF-8](https://gist.github.com/sapeurfaire/91fe247cd9523971b8fab72d50df6c2a) didn't change the output in this case.

**Comment by japhb** [comment]
That makes sense; the original screenshot wasn't broken in that particular way.  :-)
